examples/app_examples/boring_app/.gitignore

```diff
@@ -0,0 +1,10 @@
+lightning_logs
```

Comment:
not sure how this would work, but shall we rather move it on the root level?

Reply:
It's already in the root level, I think it can be deleted